development/build/index.js

```diff
@@ -4,12 +4,13 @@
 // run any task with "yarn build ${taskName}"
 //
 const livereload = require('gulp-livereload');
+const minimist = require('minimist');
```

Comment:
It looks like we already use yargs; is there a reason for using minimist over yargs? Should we be switching to minimist across the board?

Reply:
Ah, I didn't realize we were using `yargs` in the e2e tests. Interesting.

I chose `minimist` for two reasons:
1. It's dead simple, and I didn't want to go through the trouble of giving the build system the complete `yargs` treatment right now.
2. It was already in our dependency graph

Consider this a step in the right direction!

Reply:
i use yargs in lavamoat, with mixed feelings

Reply:
Cool, no worries!

Reply:
I use `yargs` in `snaps-cli`, too! I think it's pretty great, but I have bigger fish to fry right now. I'll mark this resolved.